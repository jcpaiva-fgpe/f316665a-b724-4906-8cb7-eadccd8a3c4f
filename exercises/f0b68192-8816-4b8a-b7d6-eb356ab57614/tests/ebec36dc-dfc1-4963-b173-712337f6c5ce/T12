11 15
101011001011011
111001101000011
101111101111111
010100010000000
000100000100000
001100001100000
101011100011111
111010111010100
101111101111111
010100010101000
000101000101010
