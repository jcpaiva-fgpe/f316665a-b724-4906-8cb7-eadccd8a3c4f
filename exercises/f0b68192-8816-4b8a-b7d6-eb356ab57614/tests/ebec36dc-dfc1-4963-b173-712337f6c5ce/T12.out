101101001101010
